src/routes/view-content/[guideId]/[bibleSection]/TopNavBar.svelte

```diff
@@ -38,12 +39,17 @@
         if ((!isNewTestament($currentBibleSection) || !bibleHasAlignment(currentBibleId)) && alignmentModeEnabled) {
             alignmentModeEnabled = false;
         }
+        setCurrentBibleBook(bibleSectionTitle);
     }
 
     function currentBible(bibleId: number | null) {
         return bibles.find((b) => b.id === bibleId);
     }
 
+    function setCurrentBibleBook(bibleSectionTitle: string) {
+        $currentBibleBook = bibleSectionTitle.substring(0, bibleSectionTitle.indexOf(' '));
```

Comment:
See the comment above about not using global state. Also this is not going to work on books that have multiple words like `1 Chronicles`. I'm wondering if we could get the current book name by using the `currentBibleSection` book code to index into the `bookCodesToNames`.